packages/airnode-deployer/terraform/airnode/aws/main.tf

```diff
@@ -9,7 +9,7 @@ module "initializeProvider" {
   handler            = "index.initializeProvider"
   source_dir         = var.handler_dir
   memory_size        = 768
-  timeout            = 17
+  timeout            = 20
```

Comment:
This will have conflicts with https://github.com/api3dao/airnode/pull/808 All of the workers have been combined into a single "run" worker. I don't know how this will affect memory_size and overall performance though